proto/viam/component/movementsensor/v1/movementsensor.proto

```diff
@@ -117,6 +117,8 @@ message GetOrientationRequest {
 }
 
 message GetOrientationResponse {
+  // Orientation is returned as an orientation message with 
+  // OX OY OZ dimensionalles and Theta in degrees
```

Comment:
[opt] 
```suggestion
  // OX OY OZ and Theta in degrees
```

Reply:
This isn't true, but I misspelled dimensionless.